metriq_gym/benchmarks/qml_kernel.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+from dataclasses import dataclass
+
+from qiskit import QuantumCircuit
+from qiskit.circuit import ParameterVector
+from qiskit.circuit.library import unitary_overlap
+
+from qbraid import GateModelResultData, QuantumDevice, QuantumJob
+
+from metriq_gym.benchmarks.benchmark import Benchmark, BenchmarkData, BenchmarkResult
+from metriq_gym.task_helpers import flatten_counts, flatten_job_ids
+
+
+@dataclass
+class QMLKernelData(BenchmarkData):
+    num_qubits: int
```

Comment:
Do we not also want to capture `shots` as well?

Reply:
No, because we don't use `shots` in post-processing.